Gulpfile.js

```diff
@@ -157,6 +174,30 @@ async function jsonschema() {
 
     await fs.writeJSON(destPath, schema, { spaces: 2 })
   }
+
+  // Output `types.schema.json`
+  // This 'meta' schema provides a list of type schemas as:
+  //  - an entry point for the generation of Typescript type definitions
+  //  - a lookup for all types for use in `util.ts` functions
+  const properties = {}
+  const required = []
+  for (const schema of schemas.values()) {
+    if (
+      !(schema.title && schema.$id && schema.$id.startsWith('https://stencila'))
```

Comment:
Is this comparison a bit fragile? I'd like to see it in a seperate `isStencila()` function.

Reply:
Yes it is, good point. It deals with one external file only `schema/json-schema-draft-07.schema.json` which may get removed soon anyway. So I'll leave it for now.